src/commands.ts

```diff
@@ -1021,7 +1021,8 @@ export function registerCommands(
 
 	const localUriFromReviewUri = (reviewUri: vscode.Uri) => {
 		const { path, rootPath } = fromReviewUri(reviewUri.query);
-		return vscode.Uri.joinPath(vscode.Uri.file(rootPath), path);
+		const workspaceFolder = vscode.workspace.workspaceFolders![0];
```

Comment:
Could this ever be length=0 / could this throw/

Reply:
This will only be called when there's a PR open, and a PR can only be open if there's at least one workspace folder, so that should never happen.